src/components/AddToDoInput.tsx

```diff
@@ -40,8 +39,8 @@ export function AddToDoInput(props: AddToDoInputProps) {
                         className="Todo-Description-Text"
                         id="description"
                         name="description"
-                        value={description}
-                        onChange={(e) => setDescription(e.target.value)}
+                        value={toDoContext.description}
+                        onChange={(e) => toDoContext.setDescription(e.target.value)}
```

Comment:
Unlike here, because you need to access e.target.value, you have to write the entire callback 👍